extensions/amp-story/validator-amp-story.protoascii

```diff
@@ -864,6 +864,14 @@ tags: {  # <amp-story-page-attachment> with href
       protocol: "https"
     }
   }
+  attrs: {
+    name: "cta-image"
+    value_url: {
+      # protocol: "http"  # TODO(honeybaderdontcare): readd once transformers in prod
```

Comment:
it looks like this TODO was copied from earlier; OOC, do you know anything about it (bug number, status, etc.)?

(side note: lol, "re-add". i thought for a sec this was like an intensified version of "read". like how you'd yell it to somebody who's being stubborn.)

Reply:
It's from this! https://critique-ng.corp.google.com/cl/362183596